src/current/v23.1/sso-db-console.md

```diff
@@ -6,105 +6,110 @@ docs_area: manage
 
 1. Open the [Credentials page](https://console.developers.google.com/apis/credentials) for your account at Google APIs.
 
 1. Click **+ CREATE CREDENTIALS** and select **OAuth client ID**. Specify a **web application** from the pulldown menu.
 
-1. Note the *client ID* and *client secret* of the OAuth 2.0 client. You can find these in the dialog that appears after you create the client, or in the details view for the client:
+1. Note the *client ID* and *client secret* of the OAuth 2.0 client—you will need to configure your cluster to use these values.
+
+1. Add your cluster's callback URL to the list of **Authorized redirect URIs**. On a local cluster, this will be `https://{ your cluster's domain }:8080/oidc/v1/callback`. Subsequently, when configuring your cluster, you will need to ensure that the cluster setting `server.oidc_authentication.redirect_url` has the same value.
+
+	- For a {{ site.data.products.core }} cluster, The domain is `localhost`. 
+	- For a {{ site.data.products.dedicated }} cluster, find the domain by opening the DB Console from your cluster's **Tools** tab in DB Console.
+
+## Configure your cluster to use an OIDC client and provider
 
-	<img src="{{ 'images/v23.1/google-oidc-client.png' | relative_url }}" alt="Google OAuth 2.0 client details" style="border:1px solid #eee;max-width:100%" />
+**Prerequisites**:
 
-1. Add the callback URL to the list of **Authorized redirect URIs**. On a local cluster, this will be `https://localhost:8080/oidc/v1/callback`. You will later set `server.oidc_authentication.redirect_url` to the same value.
+You must have the ability to update your cluster settings, which you can achieve in several ways. Refer to the Required Privileges section on the [`SET CLUSTER SETTING`](set-cluster-setting.html#required-privileges) page.
 
-1. Open a SQL shell to the cluster on node 1:
+### Cluster Settings
+
+You must configure the [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %} in the following table to enable SSO authentication to the DB Console. Refer to the [Update your cluster settings](#update-your-cluster-settings) section to configure your cluster settings.
+
+| Cluster Setting | Description 
+|-----------------|------
+| `server.oidc_authentication.enabled` | A Boolean that enables or disables SSO.
+| `server.oidc_authentication.client_id` | Your auth client's ID.<br>**Example:** `32789079457-g3hdfw8cbw85obi5cb525hsceaqf69unn.apps.googleusercontent.com`.
+| `server.oidc_authentication.client_secret` | Your auth client's secret.
+| `server.oidc_authentication.redirect_url` | Specifies the callback URL that redirects the user to CockroachDB after a successful authentication. This can be the address of a node in the cluster or the address of a load balancer that routes traffic to the nodes. You must append the path with `/oidc/v1/callback`. E.g., `https://{ your cluster's domain } :8080/oidc/v1/callback`.
```

Comment:
```suggestion
| `server.oidc_authentication.redirect_url` | Specifies the callback URL that redirects the user to CockroachDB after a successful authentication. This can be the address of a node in the cluster or the address of a load balancer that routes traffic to the nodes. You must append the path with `/oidc/v1/callback`.<br>**Example:** `https://{ your cluster's domain } :8080/oidc/v1/callback`.
```